src/main/java/org/folio/dataimport/util/marc/MarcRecordAnalyzer.java

```diff
@@ -40,7 +40,7 @@ private char getRecordTypeCode(JsonObject record) {
     try {
       return isMarcRecord(record) ? record.getString(LEADER_KEY).charAt(RECORD_TYPE_INDEX) : Character.MIN_VALUE;
     } catch (Exception e) {
-      LOGGER.error("I could not get a record type character form the leader: {}", e, record.getString(LEADER_KEY));
+      LOGGER.error("I could not get a record type character form the leader: {}",record.getString(LEADER_KEY), e );
```

Comment:
nice logging style)) - let's lose the "I"

Reply:
@KaterynaSenchenko Done